service-front/module/Application/src/Services/SiriusApiService.php

```diff
@@ -130,4 +130,20 @@ public function searchAddressesByPostcode(string $postcode, Request $request): a
 
         return json_decode(strval($response->getBody()), true);
     }
+
+    /**
+     * @psalm-suppress PossiblyUnusedReturnValue
+     */
+    public function abandonCase(array $data, Request $request): array
+    {
+        $response = $this->client->post('/api/v1/identity-check', [
+            'headers' => $this->getAuthHeaders($request),
+            'data' => $data
+        ]);
+
+        return [
+            'status' => $response->getStatusCode(),
+            'error' => json_decode(strval($response->getBody()), true)
+        ];
+    }
```

Comment:
This needs a Pact test (in `SiriusApiServicePactTest`) to establish a contract between the services